pkg/secrethub/fakeclient/secret.go

```diff
@@ -12,11 +12,12 @@ type SecretService struct {
 	VersionService     secrethub.SecretVersionService
 	DeleteFunc         func(path string) error
 	GetFunc            func(path string) (*api.Secret, error)
+	ReadFunc           func(path string) (*api.SecretVersion, error)
+	ReadStringFunc     func(path string) (string, error)
+	ExistsFunc         func(path string) (bool, error)
 	WriteFunc          func(path string, data []byte) (*api.SecretVersion, error)
 	ListEventsFunc     func(path string, subjectTypes api.AuditSubjectTypeList) ([]*api.Audit, error)
 	AuditEventIterator *AuditEventIterator
-
-	secrethub.SecretService
```

Comment:
Same thing as in https://github.com/secrethub/secrethub-go/pull/206 goes for removing this as well.

Reply:
Done